dldashboard/web/src/MobileIndex.js

```diff
@@ -15,6 +15,8 @@ import InteractApp from "./components/Interact/InteractApp";
 
 import "./index.css";
 
+// renders 3 panes, one for RGB, another for memory 2d, and another for chat
```

Comment:
For function level comments, use `/** ... */`
See https://google.github.io/styleguide/jsguide.html for Javascript style guide. We are not following this strictly, so use it as a loose reference.